lib/openstack/amqp/openstack_qpid_receiver.rb

```diff
@@ -82,16 +82,28 @@ def duration # :nodoc:
       durable: true
     }
   },
+  link: {
+    name: %{queue_name}
+  }
```

Comment:
The secret sauce for controlling the name of the binding queue in Qpid.

Note: the same thing will be done for Rabbit in a separate PR.